README.md

```diff
@@ -107,6 +107,8 @@ Arguments:
         (defaults to False)
     use_platform_slug: Use the NetBox platform slug for the platform attribute of a Host
         (defaults to False)
+    use_napalm: Use the Netbox platform napalm driver setting for the platform attribute
```

Comment:
Can we rename the option to `use_platform_napalm_driver`. It better describes what the option exactly does